modules/network-observability-roles-create.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-roles-create_{context}"]
+
+= Create roles for authentication and authorization
+Specify authentication and authorization configurations by defining `ClusterRole` and `ClusterRoleBinding`. You can create a YAML to define these roles.
+.Procedure
+
+. Using the web console, navigate to the Imoport icon *+*, in the top right corner. 
+. Drop your YAML file into the editor and click *Create*: 
++
+[source, yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: loki-netobserv-tenant
+rules:
+- apiGroups:
+  - 'loki.grafana.com'
+  resources:
+  - network
+  resourceNames:
+  - logs
+  verbs:
+  - 'get'
+  - 'create'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: loki-netobserv-tenant
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: loki-netobserv-tenant
+subjects:
+- kind: ServiceAccount
+  name: flowlogs-pipeline-transformer  <1>
+  namespace: netobserv
+- kind: ServiceAccount
+  name: netobserv-plugin               <2>
+  namespace: netobserv
+----
+<1> The `flowlogs-pipeline-transformer` writes to Loki.
+<2> The `netobserv-plugin` reads in the data.
+
+[NOTE]
+====
+To uninstall Loki, and Kafka if it is in use, refer to the uninstallation process that corresponds with the method you used to install Loki. You might have remaining `ClusterRoles` and `ClusterRoleBindings` or data stored in object store and persistent volume that must be removed. 
```

Comment:
Kafka section comes after this note, so we should do a separate note for kafka uninstallation in the `Installing Kafka` section.

```suggestion
To uninstall Loki, refer to the uninstallation process that corresponds with the method you used to install Loki. You might have remaining `ClusterRoles` and `ClusterRoleBindings` or data stored in object store and persistent volume that must be removed. 
```

Reply:
Addressed